fictadvisor-api/src/v2/api/services/ScheduleService.ts

```diff
@@ -457,12 +457,8 @@ export class ScheduleService {
     const data = await this.eventRepository.find({
       id: eventId,
     });
-    if (data.eventInfo) {
-      for (const eventInfo of data.eventInfo) {
-        if (eventInfo.number === index) {
-          return true;
-        }
-      }
+    if (data.eventInfo && some(data.eventInfo, "number", index)) {
```

Comment:
You can simplify this code by returning condition directly, also use single quotes for stringsas it follows correct code style.